cmd/helm/template.go

```diff
@@ -71,6 +117,7 @@ func newTemplateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 
 	f := cmd.Flags()
 	addInstallFlags(f, client, valueOpts)
+	f.StringArrayVarP(&renderFiles, "execute", "x", []string{}, "only execute the given templates")
```

Comment:
Can we change the name of the flag to something more representative of the actual behaviour? In Helm 2, I believe we were choosing which templates to pass through the rendering engine. In Helm 3, we are rendering all manifests, but we're selectively choosing which files to display on stdout.

What about `--show-only`?

Reply:
After reading your comment at https://github.com/helm/helm/issues/6633#issuecomment-540632323 - it makes sense!

Added an additional commit for the change: 74a2adf